### DISCUSSION TITLE
`no such tokenizer: PSITokenizer`

### DISCUSSION THREAD

**Discussion by vjpr** [prisma/prisma#13489] [Q&A] [answered]
_psi.schema.prisma_

```
datasource db {
  provider = "sqlite"
  url = env("PSI_DATABASE_URL")
}

generator client {
  provider = "prisma-client-js"
  output = "./client-psi"
}
```

```
Error: Error in connector: Error querying the database: Error querying the database: Error querying the database: no such tokenizer: PSITokenizer
```

**Comment by vjpr** [comment, marked as answer]
Seems to be an issue with my specific database:

https://github.com/dogsheep/dogsheep-photos/issues/16#issuecomment-623806687

https://www.sqlite.org/fts5.html

**Reply by vjpr** [reply]
Here is the reference to the tokenizer:

```
create virtual table prefix using fts5
(
    content=
    'groups',
    normalized_string,
    category
    UNINDEXED,
    tokenize =
    'PSITokenizer'
);
```

https://www.sqlite.org/fts5.html

Custom tokenizer: https://www.sqlite.org/fts5.html#tokenizers

**Reply by pantharshit00** [reply]
Can you open an issue about this?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
